### DISCUSSION TITLE
How to hide the senders name and the captions when forwarding?

### DISCUSSION THREAD

**Discussion by liechtenberg** [aahnik/tgcf#524] [Q&A]
At the run tab I see the option but turning it off and on doesn't do anything. Is there something I'm missing?

**Comment by MobCode100** [comment]
I believe the "Show Forwarded" is only able to hide the name, not caption

**Comment by liechtenberg** [comment]
Yeah but it's also not doing that

**Reply by MobCode100** [reply]
can you look at the tgcf.conf file and find `"show_forwarded_from": false,` . Is it true or false?

**Reply by liechtenberg** [reply]
![image](https://github.com/aahnik/tgcf/assets/54631194/3b91f54a-68e5-4ccd-a1bf-280284c77e16)
It is false

**Reply by MobCode100** [reply]
Weird, maybe try to restart the server

**Reply by liechtenberg** [reply]
![image](https://github.com/aahnik/tgcf/assets/54631194/dd31b430-7560-4bbe-952e-e8e06298a052)
How do I fix this? I've installed it with pip but it continues to say it. I also installed the tesseract-ocr one

**Reply by liechtenberg** [reply]
oops im dumb fixed it haha

**Reply by liechtenberg** [reply]
About the caption, there's really no way? That's a bummer

**Reply by MobCode100** [reply]
Yeah, currently there's no direct way for hiding the captions. But you still can use the replace plugin, which is a bit techy using regex expression (make sure you tick it)

To remove the captions simply use:
".+":""

Further guide: https://github.com/aahnik/tgcf/wiki/Replace-Plugin

**Reply by liechtenberg** [reply]
fixed it, thanks a lot for your help <3
